<?xml version="1.0" encoding="UTF-8"?>
    <source package="javax.management.relation">
        <class name="InvalidRelationIdException" line="2">
            <extends class="RelationException"/>
            <javadoc line="2">
                This exception is raised when relation id provided for a relation is already
                  used.                
                <since>
                    1.5                    
                </since>
            </javadoc>
            <declaration name="serialVersionUID" type="long" line="8"/>
            <javadoc line="9">
                Default constructor, no message put in exception.                
            </javadoc>
            <method name="InvalidRelationIdException" type="constructor" line="12"/>
            <javadoc line="15">
                Constructor with given message put in exception.                
                <param>
                    message the detail message.                    
                </param>
            </javadoc>
            <method name="InvalidRelationIdException" type="constructor" line="19">
                <params>
                    <param name="message" type="String"/>
                </params>
            </method>
        </class>
    </source>